traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -131,6 +131,9 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		 * 4.x API
 		 */
 
+		// Assign Multiple Server Capabilities
+		{Version: api.Version{Major: 4, Minor: 1}, Method: http.MethodPut, Path: `multiple_server_capabilities/{id}$`, Handler: server.AssignMultipleServerCapabilities, RequiredPrivLevel: auth.PrivLevelOperations, RequiredPermissions: []string{"SERVER:UPDATE", "SERVER:CREATE", "SERVER:READ", "SERVER-CAPABILITY:READ"}, Authenticated: Authenticated, Middlewares: nil, ID: 40792419258},
```

Comment:
Do you need to be able to create servers to use this route? Currently, assigning a single Capability to a single server does not require `SERVER:CREATE`.

Reply:
Nope, server is already created and thank you for the explanation.